### PR TITLE
[DEV-6532] Code Clean up (About the Data Pg Pt 1)

### PR DESCRIPTION
**Pull request by MaxwellKendall**
**High level description:**
- Updating the redux property names
- Updating local state / variables that refer to redux values
- Updating models and tests

**Technical details:**

d8d2808 & a815c07 - model updates
f9fba0c - removing FY prefix from period picker
482459c - renaming a lot of stuff

**JIRA Ticket:**
[DEV-6532](https://federal-spending-transparency.atlassian.net/browse/DEV-6532)

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
`N/A` Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
`N/A` Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer, Edge
`N/A` Verified mobile/tablet/desktop/monitor responsiveness
`N/A`Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [x] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
`N/A` [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
`N/A` [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
`N/A` Design review complete `if applicable`
`N/A` [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/components/search/visualizations/time/TimeVisualizationSection.jsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ export default class TimeVisualizationSection extends React.Component {
         <>
             <div className="tooltip__title">Download data by {capitalize(this.props.data.visualizationPeriod === 'fiscal_year' ? 'year' : this.props.data.visualizationPeriod)}</div>
             <div className="tooltip__text">
-                Download a CSV of award spending data that matches your search criteria, broken down by year. For complete download results, click on the &quot;Download&quot; button on the top right of this page.
+                Download a CSV of award spending data that matches your search criteria, broken down by {this.props.data.visualizationPeriod === 'fiscal_year' ? 'year' : this.props.data.visualizationPeriod}. For complete download results, click on the &quot;Download&quot; button on the top right of this page.
```
Comment (jonhill13) [inline src/js/components/search/visualizations/time/TimeVisualizationSection.jsx:73]: I do not believe this change should be here. @brett-varney created a warm-fix into QAT for a change relevant to this and was down merged.

**Review by jonhill13 — CHANGES_REQUESTED** [review]

File: src/js/components/search/visualizations/time/TimeVisualizationSection.jsx (hunk 1 above)
Comment (MaxwellKendall) [reply src/js/components/search/visualizations/time/TimeVisualizationSection.jsx]: True

Comment (brett-varney) [reply src/js/components/search/visualizations/time/TimeVisualizationSection.jsx]: It shouldn't be HERE, but should already be in dev, so I don't know why it thinks this is a change...

**Comment by MaxwellKendall** [comment]
This PR is really old and has a lot of weird commits. Opening #2295

**Pull request closed without merging**